xsd/netex_part_3/part3_salesTransactions/netex_fareDebit_version.xsd

```diff
@@ -0,0 +1,351 @@
+	</xsd:group>
+	<!-- ==== TRIP  DEBIT ================================================ -->
+	<xsd:element name="TripDebit" abstract="false" substitutionGroup="FareDebit_DummyType">
+		<xsd:annotation>
+			<xsd:documentation>A LOG ENTRY recording data about a debiting action for post-payment for a specific trip.</xsd:documentation>
+		</xsd:annotation>
+		<xsd:complexType>
+			<xsd:complexContent>
+				<xsd:restriction base="TripDebit_VersionStructure">
+					<xsd:sequence>
+						<xsd:sequence>
+							<xsd:group ref="DataManagedObjectGroup"/>
+						</xsd:sequence>
+						<xsd:sequence>
+							<xsd:group ref="LogEntryGroup"/>
+						</xsd:sequence>
+						<xsd:sequence>
+							<xsd:group ref="TripDebitGroup"/>
+						</xsd:sequence>
+					</xsd:sequence>
+					<xsd:attribute name="id" type="TripDebitIdType"/>
+				</xsd:restriction>
+			</xsd:complexContent>
+		</xsd:complexType>
+	</xsd:element>
+	<xsd:complexType name="TripDebit_VersionStructure">
+		<xsd:annotation>
+			<xsd:documentation>Type for TRIP DEBIT.</xsd:documentation>
+		</xsd:annotation>
+		<xsd:complexContent>
+			<xsd:extension base="FareDebit_VersionStructure">
+				<xsd:sequence>
+					<xsd:group ref="TripDebitGroup"/>
+				</xsd:sequence>
+			</xsd:extension>
+		</xsd:complexContent>
+	</xsd:complexType>
+	<xsd:group name="TripDebitGroup">
+		<xsd:annotation>
+			<xsd:documentation>Elements for TRIP DEBIT</xsd:documentation>
+		</xsd:annotation>
+		<xsd:sequence/>
+	</xsd:group>
+	<!-- ==== OTHER   DEBIT ================================================ -->
+	<xsd:element name="OtherDebit" abstract="false" substitutionGroup="FareDebit_DummyType">
+		<xsd:annotation>
+			<xsd:documentation>A LOG ENTRY recording data about a debiting action for a payment type other than a FARE PRODUCT SALE DEBIT, TRIP DEBIT, OFFENCE DEBIT or BOOKING DEBIT.</xsd:documentation>
+		</xsd:annotation>
+		<xsd:complexType>
+			<xsd:complexContent>
+				<xsd:restriction base="OtherDebit_VersionStructure">
+					<xsd:sequence>
+						<xsd:sequence>
+							<xsd:group ref="DataManagedObjectGroup"/>
+						</xsd:sequence>
+						<xsd:sequence>
+							<xsd:group ref="LogEntryGroup"/>
+						</xsd:sequence>
+						<xsd:sequence>
+							<xsd:group ref="OtherDebitGroup"/>
+						</xsd:sequence>
+					</xsd:sequence>
+					<xsd:attribute name="id" type="OtherDebitIdType"/>
+				</xsd:restriction>
+			</xsd:complexContent>
+		</xsd:complexType>
+	</xsd:element>
+	<xsd:complexType name="OtherDebit_VersionStructure">
+		<xsd:annotation>
+			<xsd:documentation>Type for OTHER DEBIT.</xsd:documentation>
+		</xsd:annotation>
+		<xsd:complexContent>
+			<xsd:extension base="FareDebit_VersionStructure">
+				<xsd:sequence>
+					<xsd:group ref="OtherDebitGroup"/>
+				</xsd:sequence>
+			</xsd:extension>
+		</xsd:complexContent>
+	</xsd:complexType>
+	<xsd:group name="OtherDebitGroup">
+		<xsd:annotation>
+			<xsd:documentation>Elements for OTHER DEBIT</xsd:documentation>
+		</xsd:annotation>
+		<xsd:sequence/>
```

Comment:
What happens here?

Reply:
TM: A LOG ENTRY recording data about a debiting action for a payment type other than a FARE PRODUCT SALE DEBIT, TRIP DEBIT, OFFENCE DEBIT or BOOKING DEBIT.